src/tests/EventPipeTracee/Program.cs

```diff
@@ -51,6 +52,21 @@ private static void TestBody(string loggerCategory)
             //Signal end of test data
             Console.WriteLine("1");
 
+            if (spinWait)
+            {
+                DateTime targetDateTime = DateTime.UtcNow.AddSeconds(10);
```

Comment:
Something I wanted to point out: on line 19, we call it spinWait10 (which, if we're waiting 10 seconds, makes sense). However, when we use it here as a method parameter, it's just spinWait (also perfectly fine). However, we then use a magic number here of 10...in the future, if we decide we want this value to be, say, 20, now the local variable in Main makes no sense (because spinWait10 now means we add 20 seconds). Maybe we just rename the one in main to be spinWait and then add a constant like SPIN_WAIT_SECONDS = 10 that we use on line 57. You can do whatever you thinks makes sense here though.

Reply:
I renamed everything to be SpinWait10 to be clear what it does and for how long.